### PR TITLE
added support in Azure app_service

### PR DESCRIPTION
**Pull request by rotemavni**

### CODE REVIEW

File: providers/azure/azure_service.go [hunk 1]
```diff
@@ -15,9 +15,19 @@
...
 	terraformutils.Service
 }
+
+func (s AzureService) GetSubscriptionID() string {
+	return s.Args["config"].(authentication.Config).SubscriptionID
+}
+
+func (s AzureService) GetAuthorizer() autorest.Authorizer {
+	return s.Args["authorizer"].(autorest.Authorizer)
+}
```
Comment (nimrodkor) [inline providers/azure/azure_service.go:33]: @rotemavni How did all the other azure services get around without these methods?

Comment (rotemavni) [reply providers/azure/azure_service.go:33]: @nimrodkor It's hard coded in all other azure services, I'm planning to refactor all of them to use these methods

Comment (nimrodkor) [reply providers/azure/azure_service.go:33]: @rotemavni Let's do it in 2 steps then, to make the contribution easier:
For now - do it like all other services. We'll merge it and contribute it
Then refactor across all Azure services - and merge + contribute it

WDYT?

Comment (rotemavni) [reply providers/azure/azure_service.go]: @nimrodkor It's ok, I removed this refactor

**Pull request merged**